docs/source/conf.py

```diff
@@ -185,4 +190,5 @@
 # We should ignore any python built-in exception, for instance
 nitpick_ignore = [
     ("py:class", "Logger"),
+    ("py:class", "numpy.float64"),
```

Comment:
Why is this needed?

Reply:
Suppresses `WARNING: py:class reference target not found: numpy.float64`.

It's odd that this isn't resolved by linking the numpy docs via `intersphinx_mapping` (which resolves similar warnings for other classes e.g. `numpy.dtype`), but seems to be necessary